### ISSUE TITLE
"Global is not defined" in React Vite app

### ISSUE THREAD

**Issue by sverdiyev** [schrodinger/fixed-data-table-2#673]
Fixed-data-table-2 does not work in Vite-React setup. 

## Expected Behavior
Application works. 

## Current Behavior
![image](https://user-images.githubusercontent.com/79407224/218130383-af8ae617-3955-4e4e-9981-aad83f97d58c.png)
Error: 

```
nativeRequestAnimationFrame.js:18 Uncaught ReferenceError: global is not defined
    at node_modules/fixed-data-table-2/internal/vendor_upstream/core/nativeRequestAnimationFrame.js (nativeRequestAnimationFrame.js:18:35)
    at __require (chunk-J43GMYXM.js?v=ed8548d7:11:50)
    at node_modules/fixed-data-table-2/internal/vendor_upstream/core/requestAnimationFramePolyfill.js (requestAnimationFramePolyfill.js:10:59)
    at __require (chunk-J43GMYXM.js?v=ed8548d7:11:50)
    at node_modules/fixed-data-table-2/internal/vendor_upstream/dom/ReactWheelHandler.js (ReactWheelHandler.js:26:61)
    at __require (chunk-J43GMYXM.js?v=ed8548d7:11:50)
    at node_modules/fixed-data-table-2/internal/FixedDataTable.js (FixedDataTable.js:24:49)
    at __require (chunk-J43GMYXM.js?v=ed8548d7:11:50)
    at node_modules/fixed-data-table-2/internal/FixedDataTableContainer.js (FixedDataTableContainer.js:24:46)
    at __require (chunk-J43GMYXM.js?v=ed8548d7:11:50)
```
## Possible Solution
Since Vite is getting traction, it would be nice to have the solution for the growing platform. If something is missing in the configuration, it would ne good to specify it in the docs. 

## Steps to Reproduce (for bugs)
1. setup vite+react+ts project
2. add fixed-data-table-2
3. use Table component

## Your Environment

Chrome: 109.0.5414.121

package.json(default + fixed-table) : 

```
{
  "name": "react-vite",
  "private": true,
  "version": "0.0.0",
  "type": "module",
  "scripts": {
    "dev": "vite",
    "build": "tsc && vite build",
    "preview": "vite preview"
  },
  "dependencies": {
    "fixed-data-table-2": "^1.2.16",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
  },
  "devDependencies": {
    "@types/fixed-data-table-2": "^0.8.8",
    "@types/react": "^18.0.24",
    "@types/react-dom": "^18.0.8",
    "@vitejs/plugin-react": "^2.2.0",
    "eslint": "8.22.0",
    "eslint-config-prettier": "^8.5.0",
    "eslint-config-react-app": "^7.0.1",
    "prettier": "^2.8.0",
    "typescript": "^4.6.4",
    "vite": "^3.2.3"
  }
}
```

vite config (default): 

```
import { defineConfig } from 'vite'
import react from '@vitejs/plugin-react'

// https://vitejs.dev/config/
export default defineConfig({
  plugins: [react()]
})
```

**Comment by pradeepnschrodinger** [comment]
@sverdiyev , hey, thanks for testing this out in Vite!

It looks like the error is coming from one of our _old_ vendors/third-party code which tries to reference `global`.
I'm not sure why, but it looks like Vite does not seem to provide or shim `global` by default.

But, FDT shouldn't even be using `global` in the first place.
We'll need to change usages from `global` to the more upto standard [`globalThis`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis).
However, I expect `globalThis` to **not** be supported in old browsers (like IE 11), so we might also need to add a polyfill for it.

I'll take a better look at this, and put up a PR for this in a day or two.

**Comment by pradeepnschrodinger** [comment]
@sverdiyev , this is fixed with #674 , and is available on [v1.2.17](https://github.com/schrodinger/fixed-data-table-2/releases/tag/v1.2.17).
Thanks again for filing the issue.

I'm closing this off but feel free to post questions here if you face any trouble.
